packages/atomic/cypress/integration/facets-v1/facet-common-assertions.ts

```diff
@@ -0,0 +1,134 @@
+import {TestFixture} from '../../fixtures/test-fixture';
+import {ComponentSelector, should} from '../common-assertions';
+
+export interface FacetsSelector extends ComponentSelector {
+  wrapper: () => Cypress.Chainable<JQuery<HTMLElement>>;
+  labelButton: () => Cypress.Chainable<JQuery<HTMLElement>>;
+  values: () => Cypress.Chainable<JQuery<HTMLElement>>;
+  selectedCheckboxValue: () => Cypress.Chainable<JQuery<HTMLElement>>;
```

Comment:
selectedCheckboxValue/idleCheckboxValue/selectedLinkValue/idleLinkValuevalueLabel aren't common to all the facets

Reply:
Maybe selectedValue or idleValue would be better as a generalization

Reply:
Or maybe divided the selectors into BaseFacetSelector / FacetWithCheckboxSelector / FacetWithLinkSelector